content/docs/guides/branch-restore.md

```diff
@@ -252,48 +235,15 @@ curl --request POST \ // [!code word:br-damp-smoke-91135977]
-
-    This makes the selection for Time Travel Assist. Notice the updated fields above the SQL editor show the **branch** and **timestamp** you just selected.
-    ![Time travel assist](/docs/guides/time_travel_assist.png)
-  
-1. Check that you have the right database selected to run your query against. Use the database selector under the SQL editor to switch to a different database for querying against.
-1. Write your read-only query in the editor, then click **Query at timestamp** to run the query. You don't have to include time parameters in the query; the query is automatically targeted to your selected timestamp.
-
-If your query is successful, you will see a table of results under the editor.
-
-Depending on your query and the selected timestamp, instead of a table of results, you might see different error messages like:
-| Error                | Explanation             |
-|----------------------|-------------------------|
-| If you query a timestamp in the future         | Console request failed with 400 Bad Request: timestamp [timestamp] is in the future, try an older timestamp |
-| If you query a timestamp from before your project was created | Console request failed with 400 Bad Request: parent timestamp [timestamp] is earlier than the project creation timestamp [timestamp], try a more recent timestamp |
-| If you query from earlier than your history retention window | Console request failed with 400 Bad Request: timestamp [timestamp] recedes your project's history retention window of 168h0m0s, try a more recent timestamp |
-
-Adjust your selected timestamp accordingly.
-
 ## Billing considerations
 
-There are minimal impacts to billing from the branch restore and Time Travel Assist features.
-
-### Restore from history (billing)
-
-Restoring a branch to its own history adds to your number of branches &#8212; due to the restore_backup branch &#8212; but since they do not have any compute endpoint attached, they do not add to any consumption costs.
-
-### Time travel Assist (billing)
-
-The ephemeral endpoints used to run your Time Travel Assist queries do contribute to your consumption usage totals for the billing period, like any other active endpoint that consumes resources.
-
-A couple of details to note:
+There are minimal impacts to billing from the branch restore and Time Travel Assist features:
 
-- The endpoints are shortlived. They are suspended 10 seconds after you stop querying.
-- Since these endpoints are created according to your default compute size (which applies to all new branch computes you create),  you may want to reduce this default if you're performing a lot of time-travel queries for troubleshooting.
+- **Branch Restore** &#8212;  &#8212; The backups created when you restore a branch do add to your total number of branches, but since they do not have any compute endpoint attached they do not add to any consumption costs.
```

Comment:
```suggestion
- **Branch Restore** &#8212;  The backups created when you restore a branch do add to your total number of branches, but since they do not have any compute endpoint attached they do not add to any consumption costs.
```